Vision/README.md

```diff
@@ -1,51 +1,47 @@
 # The W3C Vision: An Emphasis on Values
 
 ## Introduction
-The World Wide Web was conceived more than 25 years ago 
+The World Wide Web was originally conceived 
 as a tool for sharing information. 
-It has become much more than that; 
-it is a fundamental part of the lives of much of humanity, 
-enabling access to information, education, 
+It has evolved into a fundamental part of humanity, 
+providing access to knowledge, education, 
 commerce and shopping, social experiences, 
-civic functions, entertainment, and more. 
+civic duties, entertainment, and more.
 
-The Web is a force for good; 
-indeed, it has catalyzed major social changes. 
-At the same time, the Web's phenomenal success 
+The Web has been a force for good, 
```

Comment:
```suggestion
The Web was foreseen as a force for good, 
```

Reply:
I'm not sure what the goal of this language change is?  It's less translatable, and less direct.

Reply:
The change aims to be explicit that there was a conscious plan for the web to "improve" the world - it wasn't built as a random technological artifact that might do good or might just be toxic.

It is also motivated by the fact that "A force for good" is an incredibly subjective term. Using it as if it were a simple reality seems likely to make it hard to ever get broad agreement around an operable version of this document.

Reply:
I think that intention here is to state that web was always intended to be a force for good. We want to state the intention.

Reply:
Then say "The web is, and was designed to be, a force for good". But I think that assuming the intent and stating the fact is better than stating the intent and leaving whether it's happened in doubt.